nemo/utils/exp_manager.py

```diff
@@ -333,24 +333,15 @@ def exp_manager(trainer: 'pytorch_lightning.Trainer', cfg: Optional[Union[DictCo
         resume_if_exists=cfg.resume_if_exists,
     )
 
-    if cfg.resume_if_exists:
```

Comment:
Id suggest keeping `if resume_if_exists` outside as well as passing it as an arg. Its the first bool flag that checks if you even want to resume or not.

Reply:
My understanding was that `resume_if_exists` only applies to the checkpoints in the log dir, and `resume_from_checkpoint` is outside that. So this will enforce that `resume_if_exists` must be set to use any kind of checkpoint, either from the log dir or manually specified with resume from, is that correct?